amazon/no-nap/entrypoint.sh

```diff
@@ -97,12 +96,10 @@ fi
 
 wait_term()
 {
-    wait ${nginx_pid}
-    trap - TERM
-    echo "wait for nginx to stop..."
-    wait ${nginx_pid}
+    wait ${agent_pid}
```

Comment:
Could you explain why you replaced 
```
    trap - TERM	    handle_term
    echo "wait for nginx to stop..."	
    wait ${nginx_pid}
```
 with just `handle_term`? 
Now it will send TERM signal to agent and nixing twice and docker will not wait for actually gracefully shutdown of the agent/nginx process.

Reply:
updated, here is the output if I stop the agent process:
```
2021/01/21 20:02:20 [notice] 8#8: signal 3 (SIGQUIT) received from 1, shutting down
waiting for nginx to stop...
2021/01/21 20:02:20 [notice] 16#16: gracefully shutting down
2021/01/21 20:02:20 [notice] 14#14: gracefully shutting down
2021/01/21 20:02:20 [notice] 14#14: exiting
2021/01/21 20:02:20 [notice] 13#13: gracefully shutting down
2021/01/21 20:02:20 [notice] 14#14: exit
2021/01/21 20:02:20 [notice] 13#13: exiting
2021/01/21 20:02:20 [notice] 15#15: gracefully shutting down
2021/01/21 20:02:20 [notice] 15#15: exiting
2021/01/21 20:02:20 [notice] 13#13: exit
2021/01/21 20:02:20 [notice] 15#15: exit
2021/01/21 20:02:20 [notice] 16#16: exiting
2021/01/21 20:02:20 [notice] 16#16: exit
2021/01/21 20:02:20 [notice] 8#8: signal 17 (SIGCHLD) received from 15
2021/01/21 20:02:20 [notice] 8#8: worker process 13 exited with code 0
2021/01/21 20:02:20 [notice] 8#8: worker process 14 exited with code 0
2021/01/21 20:02:20 [notice] 8#8: worker process 15 exited with code 0
2021/01/21 20:02:20 [notice] 8#8: worker process 16 exited with code 0
2021/01/21 20:02:20 [notice] 8#8: exit
controller-agent process has stopped, exiting.
```